odoo/tools/config.py

```diff
@@ -244,18 +257,18 @@ def _build_cli(self):
 
         # WEB
         group = optparse.OptionGroup(parser, "Web interface Configuration")
-        group.add_option("--db-filter", dest="dbfilter", my_default='', metavar="REGEXP",
+        group.add_option("--db-filter", dest="dbfilter", my_default='', metavar="REGEXP", env_name='ODOO_DB_FILTER',
```

Comment:
```suggestion
        group.add_option("--db-filter", dest="dbfilter", my_default='', metavar="REGEXP",
```
one day we'll do an alias to also allow `db_filter` in the config, and hence `ODOO_DB_FILTER`